src/routes/summary/index.svelte

```diff
@@ -80,16 +86,32 @@
 			<SummaryBox items={unquizzed} fill={false} truncateAt={15} />
 		</section>
 	{/if}
+
+	{#if [...correct, ...incorrect].length > 0}
+		<section>
+			<h2>Kana times</h2>
+			<SummaryBox
+				time
+				items={[...correct, ...incorrect].sort(
+					(a, b) => b.duration - a.duration
+				)}
+				truncateAt={10}
+			/>
+		</section>
+	{/if}
 </div>
 
 <MenuBar class="glass-morphism">
 	<div class="menu content-width content-padding center">
 		{#if unquizzed.length > 0}
-			<p>There are {unquizzed.length} left to quiz. Finish up?</p>
+			<p>
+				There {unquizzed.length === 1 ? "is" : "are"}
+				{unquizzed.length} left to quiz. Finish up?
+			</p>
 		{/if}
 		<div class="menu-items">
 			{#if unquizzed.length > 0}
-				<Button href="/session">
+				<Button href="/session" on:click={() => quiz.refreshTime()}>
```

Comment:
Same as other comment. There are too many edge cases for me to be comfortable with this solution